source/SH3/system/glprogram.cpp

```diff
@@ -51,21 +53,24 @@ GLint sh3_glprogram::Load(const std::string& name)
 
 void sh3_glprogram::Bind()
 {
-
+    if(programID != 0)
+        glUseProgram(programID);
 }
 
 void sh3_glprogram::Unbind()
 {
-
+    glUseProgram(0); // The 'correct' way to unbind a shader, but apparently undefined?!?!
```

Comment:
Yeah that sounds like a good idea. Maybe even a checkered square like the Source engine haha.